packages/filecoin-actor-utils/src/utils/generic.ts

```diff
@@ -7,18 +9,33 @@ import { DataType, Type } from '../types'
  * @param value the value to add to the descriptor
  */
 export const describeDataType = (dataType: DataType, value: any) => {
+  // Check special types by name
+  switch (dataType.Name) {
+    case 'Address':
+      describeAddress(dataType, value)
+      return
+  }
```

Comment:
Can an Address not just be a `dataType`?

Reply:
https://github.com/glifio/filecoin-descriptors/blob/c45bcc4acc7d7cf403d5a61d6b30e7171f7fed86/reflection.go#L20 i see